src/components/Calculator.js

```diff
@@ -1,35 +1,39 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prop-types */
 import React from 'react';
 import './Calculator.css';
+import Output from './Output';
 
 class Calculator extends React.Component {
-  constructor() {
-    super();
-    this.state = {};
-  }
-
   render() {
+    const { total, next, operation } = this.props;
     return (
       <div className="calc-grid">
-        <div className="output" data-current-operation>0</div>
-        <button type="button" data-clear-all>AC</button>
-        <button type="button" data-change-sign>+/-</button>
-        <button type="button" data-percentage>%</button>
-        <button type="button" id="orange-background" data-operation>÷</button>
-        <button type="button" data-number>7</button>
-        <button type="button" data-number>8</button>
-        <button type="button" data-number>9</button>
-        <button type="button" id="orange-background" data-operation>×</button>
-        <button type="button" data-number>4</button>
-        <button type="button" data-number>5</button>
-        <button type="button" data-number>6</button>
-        <button type="button" id="orange-background" data-operation>-</button>
-        <button type="button" data-number>1</button>
-        <button type="button" data-number>2</button>
-        <button type="button" data-number>3</button>
-        <button type="button" id="orange-background" data-operation>+</button>
-        <button type="button" className="span-two" data-number>0</button>
-        <button type="button" data-number>.</button>
-        <button type="button" id="orange-background" data-equals>=</button>
+        <Output
+          total={total}
+          operation={operation}
+          next={next}
+        />
+        <button type="button" onClick={() => this.props.clickHandler('AC')} data-clear-all>AC</button>
+        <button type="button" onClick={() => this.props.clickHandler('+/-')} data-change-sign>+/-</button>
+        <button type="button" onClick={() => this.props.clickHandler('%')} data-percentage>%</button>
+        <button type="button" onClick={() => this.props.clickHandler('÷')} id="orange-background" data-operation>÷</button>
+        <button type="button" onClick={() => this.props.clickHandler('7')} data-number>7</button>
+        <button type="button" onClick={() => this.props.clickHandler('8')} data-number>8</button>
+        <button type="button" onClick={() => this.props.clickHandler('9')} data-number>9</button>
+        <button type="button" onClick={() => this.props.clickHandler('×')} id="orange-background" data-operation>×</button>
```

Comment:
- [ ] <b>REQUIRED:</b> Multiplication is not working. it is throwing the following error:
![Screen Shot 2022-02-11 at 14 31 35](https://user-images.githubusercontent.com/45217364/153592081-0d41b434-f352-4869-838d-8f0ed1cf1d28.png)
Kindly address the issue and fix it.